log/log.go

```diff
@@ -39,10 +41,64 @@ func (f levelFlag) Set(level string) error {
 	return nil
 }
 
+func setSyslogFormatter(appname, facility string) error {
+	if appname != "" || facility != "" {
```

Comment:
This is still wrong. If either of them is set we'll return early here. It needs to be `==` in both comparisons.

A test might make sense at this point ;)